crates/ironrdp/Cargo.toml

```diff
@@ -15,17 +15,21 @@ categories.workspace = true
 doctest = false
 test = false
 
+[dependencies]
+ironrdp-pdu = { workspace = true, optional = true }
+ironrdp-connector = { workspace = true, optional = true }
+ironrdp-session = { workspace = true, optional = true }
+ironrdp-graphics = { workspace = true, optional = true }
+ironrdp-input = { workspace = true, optional = true }
+ironrdp-tokio = { workspace = true, optional = true }
+ironrdp-tls = { workspace = true, features = ["rustls"], optional = true }
+
 [features]
 default = ["pdu", "connector", "session"]
 pdu = ["dep:ironrdp-pdu"]
 connector = ["dep:ironrdp-connector"]
 session = ["dep:ironrdp-session"]
 graphics = ["dep:ironrdp-graphics"]
 input = ["dep:ironrdp-input"]
```

Comment:
Curiosity/Nit: is there a specific reason for moving the `[[dependencies]]` section before the `[[features]]` one?
[Most](https://github.com/snapview/tokio-tungstenite/blob/dfe10bb2a84110e221c0b60bbdd4ac9bd1a92885/Cargo.toml#L19) projects I know are keeping feature list [before](https://github.com/tokio-rs/tokio/blob/c84d0a14b189c45da8f5e963fd3a83790ec92f8e/tokio/Cargo.toml#L24), like we currently do

Reply:
My thinking was that because the "dep:<crate-name>" features are being generated by the optional dependencies, it made sense for dependencies to go before features. I wasn't aware there was a common convention though, let's just stick with that.

ead3fe6060fb22df28f2867acdf25fcaebf8aa2b